frontend/io-webapp/src/endpointHandlers.js

```diff
@@ -31,3 +31,26 @@ export const getParticipantsHandler = (e, state, dispatch) => {
         });
     })();
   };
+
+export const getSessionHistoryHandler = (e, state, dispatch) => {
+    e.preventDefault();
+
+    (async () => {
+        await fetch("http://localhost:8080/session/history", {
```

Comment:
i tutaj nie powinno być localhost, tylko z parametrem z adresem serwera

Reply:
Jasne, już to zmieniłem ^^